io/post_fv3.F90

```diff
@@ -592,6 +593,7 @@ subroutine set_postvars_fv3(wrt_int_state,grid_id,mype,mpicomp)
 !
       integer i, ip1, j, l, k, n, iret, ibdl, rc, kstart, kend
       integer i1,i2,j1,j2,k1,k2
+      integer landsfcmdl
```

Comment:
@JiliDong-NOAA Do you think this allocation might be unnecessary?

Reply:
I am not sure about that. Just to be safe I guess.